standalone-e2e-test/src/test/java/configuration/RestAssuredTestConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@ComponentScan("config")
```

Comment:
I don't think that you have any root package called config, there spring won't find any bean for you. You can remove it.